### ISSUE TITLE
Enhancement - SSM Version Support

### ISSUE THREAD

**Issue by terrancej** [helmfile/vals#25]
I would like to be able to choose the version of a SSM parameter. Such that:
`ref+awsssm://PATH/TO/PARAM[?region=REGION][?version=VERSION]`

Creating a PR for this

**Comment by terrancej** [comment]
I'll update the README.

@mumoshu thoughts on
https://github.com/variantdev/vals/blob/a611e82bc24b845fa97953bfdc728749bc06083b/pkg/providers/ssm/ssm.go#L59 &
https://github.com/variantdev/vals/blob/a611e82bc24b845fa97953bfdc728749bc06083b/pkg/providers/ssm/ssm.go#L99

I was testing on my local Helmfile with the updated vals and I get the following. I'd love to suppress the output. I see that some other providers don't have a log line but maybe there is value in being able to see the value being pulled. I just don't want credentials in my CI logs 
```
# tail from helmfile template
...
Deleting outdated charts

foo.yaml: basePath=.

hook[prepare] logs | namespace/foo unchanged
hook[prepare] logs | 
SSM: successfully retrieved key=bar
```

**Comment by mumoshu** [comment]
@terrancej Thanks! I'm trying to isolate the cause. Are you giving helmfile a `--log-level=debug` flag?

**Comment by mumoshu** [comment]
I've realized that `SSM: successfully retrieved key=bar` is logged vals and I made it togglable via a variable in https://github.com/helmfile/vals/commit/6ef94b083014e69a989ad76ffcd01fa4e76c0fa4.
Perhaps we can set the variable from helmfile or add a field to the vals runtime struct for togglign and set it from helmfile to suppress the logs.

**Comment by mumoshu** [comment]
The fix is going to be live via v0.23.0!
